b2h5py/tests/test_b2dataset.py

```diff
@@ -0,0 +1,37 @@
+"""Test using B2Dataset to enable fast access to Blosc2 compressed dataset.
+"""
+
+import h5py
+import hdf5plugin
+import numpy as np
+import pytest
```

Comment:
Please note that plain `b2h5py` tests don't use pytest, only vanilla `unittest` to avoid the extra dependency (which is only needed by the `h5py-test` extra feature to run h5py tests). You can see that this did cause a CI check to fail: <https://github.com/Blosc/b2h5py/actions/runs/7274572901/job/19860011744?pr=1>.

I think this should be rewritten for `unittest`.

Reply:
Tests rewritten as a `TestCase`, CI passes